modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,212 @@
+
+.Procedure
+
+. Check your AWS account for existing roles and policies:
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
+
+. Create the Terraform files locally by using the following codes samples:
+.. Copy this template into a file  named `main.tf`. Running the main.tf file calls the Red Hat Cloud Services Terraform provider, which allows you to use OpenShift services with Terraform.
++
+[source,terraform]
+----
+#
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.20.0"
+    }
+    rhcs = {
+      version = ">= 1.1.0"
+      source  = "terraform-redhat/rhcs"
+    }
+  }
+}
+
+provider "rhcs" {
+  token = var.token
+  url   = var.url
+}
+
+data "rhcs_policies" "all_policies" {}
+
+data "rhcs_versions" "all" {}
+
+module "create_account_roles" {
+  source  = "terraform-redhat/rosa-sts/aws"
+  version = "0.0.12"
+
+  create_operator_roles = false
+  create_oidc_provider  = false
+  create_account_roles  = true
+
+  account_role_prefix    = var.account_role_prefix
+  ocm_environment        = var.ocm_environment
+  rosa_openshift_version = var.openshift_version
+  account_role_policies  = data.rhcs_policies.all_policies.account_role_policies
+  operator_role_policies = data.rhcs_policies.all_policies.operator_role_policies
+  all_versions           = data.rhcs_versions.all
+  path                   = var.path
+  tags                   = var.tags
+}
+----
+
+.. Copy this template into a file  named `output.tf`. In the `output.tf` file, the account role prefix structure is defined. This output definition allows you to specify how the various generated roles are constructed. 
++
+[source,terraform]
+----
+output "account_role_prefix" {
+  value = module.create_account_roles.account_role_prefix
+}
+----
+
+.. Copy this template into a file  named `variables.tf`. You can provide your own variables in the default fields for the account-role creation variables. 
++
+[IMPORTANT]
+====
+Do not include your OCM token in this file if it is not stored in a safe location.
+====
++
+[source,terraform]
+----
+variable "ocm_environment" {
+  type    = string
+  default = "production"
+}
+
+variable "openshift_version" {
+  type = string
+  default = ""
+}
+
+variable "account_role_prefix" {
+  type    = string
+  default = ""
+}
+
+variable "token" {
+  type = string
+}
```

Comment:
Please remove this "token" variable and add instructions on how to set this as environment variable
User should export the environment variable:
`export OCM_TOKEN="<the token>"`
This way is safer